framework/include/problems/FEProblemBase.h

```diff
@@ -220,24 +220,23 @@ class FEProblemBase : public SubProblem, public Restartable
    * @param abstol         Absolute residual convergence tolerance
    * @param nfuncs         Number of function evaluations
    * @param max_funcs      Maximum Number of function evaluations
-   * @param initial_residual_before_preset_bcs      Residual norm prior to imposition of preset BCs
-   * values on solution vector
+   * @param pre_initial_residual      Residual norm prior to imposition of solution modifying
```

Comment:
This spacing is weird. I would just do a single space between all the params and their documentation strings